client/signup/steps/design-picker/index.jsx

```diff
@@ -63,13 +115,43 @@ class DesignPickerStep extends Component {
 				theme={ this.props.isReskinned ? 'light' : 'dark' }
 				locale={ this.props.locale } // props.locale obtained via `localize` HoC
 				onSelect={ this.pickDesign }
+				onPreview={ this.previewDesign }
 				className={ classnames( {
 					'design-picker-step__is-large-thumbnails': this.props.largeThumbnails,
 				} ) }
 			/>
 		);
 	}
 
+	renderDesignPreview() {
+		const {
+			signupDependencies: { siteSlug },
+			locale,
+			translate,
+		} = this.props;
+
+		const { selectedDesign } = this.state;
+		const previewUrl = getDesignUrl( selectedDesign, locale );
+
+		return (
+			<WebPreview
+				className="design-picker__web-preview"
+				showPreview
+				isContentOnly
+				showClose={ false }
+				showEdit={ false }
+				externalUrl={ siteSlug }
+				previewUrl={ previewUrl }
+				loadingMessage={ translate( '{{strong}}One moment, please…{{/strong}} loading your site.', {
```

Comment:
I think we would usually use more succinct language, what do you think @ollierozdarz 

I'd probably just go with:

> loading your site

Reply:
I just re-use the wording from https://github.com/Automattic/wp-calypso/blob/f4b8378eaa8772d0f7dc1ef2fd2c20e7741adf52/client/my-sites/preview/main.jsx#L182 😅